nml/expression/binop.py

```diff
@@ -187,22 +217,21 @@ def is_boolean(self):
         return self.op.returns_boolean
 
     def __eq__(self, other):
-        return other is not None and isinstance(other, BinOp) and self.op == other.op and self.expr1 == other.expr1 and self.expr2 == other.expr2
+        return (
+            other is not None
+            and isinstance(other, BinOp)
+            and self.op == other.op
+            and self.expr1 == other.expr1
+            and self.expr2 == other.expr2
+        )
 
     def __ne__(self, other):
         return not self.__eq__(other)
 
     def __hash__(self):
         return hash((self.op, self.expr1, self.expr2))
 
```

Comment:
really? In all places lists are stretched to single lines for entries here the reverse? --> broken alignment

Reply:
Because everything fits on one line, so it squashes everything together :)